app/models/miq_request_workflow.rb

```diff
@@ -365,7 +370,6 @@ def self.parse_ws_string(text_input, options = {})
   def self.parse_request_parameter_hash(parameter_hash, options = {})
     parameter_hash.each_with_object({}) do |param, hash|
       key, value = param
-      next if value.blank?
```

Comment:
should be ok, if callers in REST intentionally set a value to "", they may expect it to make it down to the provisioning layers.